src/app_charts/prometheus/prometheus-robot.values.yaml

```diff
@@ -140,6 +145,11 @@ kubelet:
 nodeExporter:
   enabled: true
   serviceMonitor:
+    metricRelabelings:
+    # Drop high cardinality metrics from node exporter.
+    - action: drop
+      regex: "node_cpu_seconds_total"
+      sourceLabels: [ __name__ ]
```

Comment:
Is this also labeled with e.g. 'core' to cause high cardinality? Is this used in the performance grafana dashboards? (The one we print after inctl app start).

Reply:
The dashboard is based on `node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate`. Based on the name, I think that's not based on this metric so will hopefully survive, but I'm not sure how to check which rules/dashboards will be broken by this change.

On this metric specifically, I was surprised it's so big, and I think your intuition is right. It's 11k metrics on giza-workcells, ranging from 48 to 944 per cluster, where xfa-box-32 is the worst. `cpu` seems to be the worst label (0..109) but `instance` and `pod` are slightly noisy (random IPs / node exporter pod IDs, but correlated with the cluster/node). It would be tempting to sum without `cpu`, `instance`, and `pod` if we want to keep this metric - but I don't know how easy that is to do in the relabelling config - and I don't want to block this change since Prometheus federation doesn't work right now.